src/sentry/seer/issue_summary.py

```diff
@@ -233,6 +233,44 @@ def _get_trace_connected_issues(event: GroupEvent) -> list[Group]:
     return connected_issues
 
 
+def _run_automation(
+    group: Group,
+    user: User | RpcUser | AnonymousUser,
+    event: GroupEvent,
+    source: str,
+):
+    if features.has(
+        "organizations:trigger-autofix-on-issue-summary", group.organization, actor=user
+    ):
+        with sentry_sdk.start_span(op="ai_summary.generate_fixability_score"):
+            try:
+                issue_summary = _generate_fixability_score(group.id)
+            except Exception:
+                logger.exception("Error generating fixability score", extra={"group_id": group.id})
+                return
+
+        if issue_summary.scores.fixability_score is not None:
+            group.update(seer_fixability_score=issue_summary.scores.fixability_score)
+
+        if issue_summary.scores.is_fixable:
```

Comment:
It looks like based on the data type that `scores` can also be `None`. I'm not sure if that's true in practice though. 

Probably we should update the data type to make this not null, or if it can be null, check for None here too. I'm ok pushing that to a separate pr since it was already a problem here.

Reply:
ah good catch, I added an early return in case it's None to be safe